### ISSUE TITLE
Unable to set schema compatibility level to forward_transitive

### ISSUE THREAD

**Issue by paveldanichev** [bitnami/containers#67127]
### Name and Version

bitnami/schema-registry:7.6.1

### What architecture are you using?

arm64

### What steps will reproduce the bug?

Start container of `bitnami/schema-registry:7.6.1` with the environment variable  `SCHEMA_REGISTRY_AVRO_COMPATIBILY_LEVEL=forward_transitive`
This option is documented here: https://hub.docker.com/r/bitnami/schema-registry/

I use docker-compose. This is the basic setup (part of the YAML file):
```yaml
  schema-registry:
    image: bitnami/schema-registry:7.6.1
    container_name: schema-registry
    ports:
      - "8081:8081"
      - "8082:8082"
    volumes:
      - type: bind
        source: /home/ubuntu/Documents/kafka/schema-registry-data
        target: /bitnami
    depends_on:
      - kafka
    environment:
      - SCHEMA_REGISTRY_DEBUG=true
      - SCHEMA_REGISTRY_KAFKA_BROKERS=PLAINTEXT://kafka:9092
      - SCHEMA_REGISTRY_HOST_NAME=schema-registry
      - SCHEMA_REGISTRY_LISTENERS=http://0.0.0.0:8081
      - SCHEMA_REGISTRY_AVRO_COMPATIBILY_LEVEL=forward_transitive
    networks:
      - kafka-network
```

### What is the expected behavior?

The container should start.
The default compatibility level as returned by `curl localhost:8081/config` should be `forward_transitive`.

### What do you see instead?

The container exits promptly after starting.
Docker logs contain this error:
```
schema-registry 09:13:35.41 INFO  ==> ** Starting Schema Registry setup **
schema-registry 09:13:35.42 INFO  ==> Validating settings in SCHEMA_REGISTRY_* env vars
schema-registry 09:13:35.43 ERROR ==> The allowed values for SCHEMA_REGISTRY_AVRO_COMPATIBILY_LEVEL are: none, backward, forward or full
```

### Additional information

In addition to `forward_transitive` not included as one of the legal values for `SCHEMA_REGISTRY_AVRO_COMPATIBILY_LEVEL`, which looks like it is handled here: https://github.com/bitnami/containers/blob/913d85fcaf8b8e7a7a719a6a0bb5bd2aff179928/bitnami/schema-registry/7.6/debian-12/rootfs/opt/bitnami/scripts/libschemaregistry.sh#L182

according to https://docs.confluent.io/platform/current/schema-registry/installation/config.html
the parameter `avro.compatibility.level` is deprecated and should be replaced by `schema.compatibility.level`.

**Comment by rafariossaa** [comment]
Hi,
  Could you give `Release 7.6.1-debian-12-r4` a try ?
  We have added the missing value to [the validation](https://github.com/bitnami/containers/blob/4a7d93840131d7fdcce090c7c64baf138d0dd7cb/bitnami/schema-registry/7.6/debian-12/rootfs/opt/bitnami/scripts/libschemaregistry.sh#L182).

**Comment by paveldanichev** [comment]
Hi,
I pulled `bitnami/schema-registry:7.6.1-debian-12-r4` and the validation of `SCHEMA_REGISTRY_AVRO_COMPATIBILY_LEVEL=forward_transitive` passes now,
_but this setting now has no effect_.

I tried to investigate and found the following:
The ending of the file `/opt/bitnami/schema-registry/etc/schema-registry/schema-registry.properties` looks like this:
```
## The protocol used while making calls between the instances of Schema Registry
##
inter.instance.protocol = http

## The Avro compatibility type
##
avro.compatibility.level = backward

## Enable debug logs
##
debug = true

schema.compatibility.level = forward_transitive
```
The setup added `schema.compatibility.level = forward_transitive` but also left there `avro.compatibility.level = backward` and the tatter seems to have precedence.
I manually commented out `avro.compatibility.level = backward`, restarted the container, and this way got the expected result of
`curl localhost:8081/config`
`{"compatibilityLevel":"FORWARD_TRANSITIVE"}`

**Comment by rafariossaa** [comment]
Hi,
  I missed that.
  Could you check `7.6.1-debian-12-r5` ?

**Comment by paveldanichev** [comment]
Issue resolved in `7.6.1-debian-12-r5`.
